src/materials/ShaderMaterial.js

```diff
@@ -37,7 +37,8 @@ class ShaderMaterial extends Material {
 			fragDepth: false, // set to use fragment depth values
 			drawBuffers: false, // set to use draw buffers
 			shaderTextureLOD: false, // set to use shader texture LOD
-			clipCullDistance: false // set to use vertex shader clipping
+			clipCullDistance: false, // set to use vertex shader clipping
+			multiDraw: false // set to use vertex shader multi_draw / enable gl_DrawID
```

Comment:
A new entry should be documented in: https://threejs.org/docs/index.html#api/en/materials/ShaderMaterial.extensions

Reply:
added